JetNews/app/src/main/java/com/example/jetnews/ui/components/AppNavRail.kt

```diff
@@ -77,7 +76,7 @@ fun AppNavRail(
 ) {
     JetnewsNavRail(
         header = {
-            JetnewsIcon(Modifier.statusBarsPadding().padding(top = 8.dp))
```

Comment:
This causes the `JetNews` icon to be obscured by the status bar when on the interests screen.

Reply:
True! Good catch, this is one of the things that are better handled in the other PR. Fixing it here, just in case